src/ServiceBus.AttachmentPlugin/AzureStorageAttachmentConfiguration.cs

```diff
@@ -10,16 +10,23 @@ public class AzureStorageAttachmentConfiguration
         /// <param name="connectionString"></param>
         /// <param name="containerName"></param>
         /// <param name="messagePropertyToIdentifyAttachmentBlob"></param>
+        /// <param name="sasUri"></param>
+        /// <param name="sasTokenValidInSeconds"></param>
         /// <param name="messageMaxSizeReachedCriteria">Default is always use attachments</param>
         public AzureStorageAttachmentConfiguration(string connectionString,
             string containerName = "attachments",
             string messagePropertyToIdentifyAttachmentBlob = "$attachment.blob",
+            string sasUri = "$attachment.sas.uri",
+            long sasTokenValidInSeconds = 0,
```

Comment:
Default SAS token should be valid for more than 0 seconds. Let's assume 7 days of time as a default and document that.

Reply:
Default behavior is to not use SAS uri at all. Would you like it changed to use SAS uri every time with a default of 7 days?

Reply:
I meant "default _for_ SAS token should be valid for more than 0 seconds".
The default behavior should stay as-is. SAS URI based approach should be an opt-in.

Reply:
BTW, `sasUri` property for `AzureStorageAttachmentConfiguration` is more `messagePropertyToIdentifyBlobSasUri` than `sasUri`.